### PR TITLE
Add Rapyuta Robotics

### PR DESCRIPTION
**Pull request by khatribharat**

### CODE REVIEW

**Comment by khatribharat** [comment]
@poteto Resolved the merged conflict.

**Comment by khatribharat** [comment]
Hey, I don't see any conflicts. And don't see any any requested changes.

File: README.md [hunk 1]
```diff
@@ -33,7 +33,6 @@ Please open a PR to be added.
 - [CASHLINK](https://cashlink.de) | Frankfurt, Germany | Skype/phone interview, take-home project
 - [Canva](https://about.canva.com/careers/) | Sydney, Australia | Takehome project, then a pair-programming onsite (or Skype) round.
 - [Centroida](https://centroida.co/contact.html) | Sofia, Bulgaria | Series of interviews, pair programming and take-home projects
-- [Chargify](https://chargify.com/jobs) | San Antonio, TX / Remote | Take-home project & pair on a problem similar to daily work
```
Comment (poteto) [inline README.md]: Please add this back!

**Comment by poteto** [comment]
Thanks!

**Pull request merged**